### ISSUE TITLE
Issues upgrading Flux using bootstrap github command

### ISSUE THREAD

**Issue by slacksach** [fluxcd/flux2#1235]
### Describe the bug

When attempting to upgrade an existing Flux 0.9.1 installation to 0.12.0 using the exact same bootstrap command as 0.9.1 was installed with an error is received:

```
sync path configuration ("./overlays\\test\\subfolder") would overwrite path ("./overlays/test/subfolder") of existing 
Kustomization
```

### To Reproduce

Run flux bootstrap github with a path containing subfolders. `overlays/test/subfolder`
Run flux bootstrap github again with the same path. `overlays/test/subfolder`

The GitHub repository is updated and manifests generated as expected, however, the error above is returned at the sync stage.
Appears that some additional escaping is taking place.

**Comment by hiddeco** [comment]
Found the issue, at present waiting on my Windows VM download to finish so I can confirm the patch. Thanks for reporting :sunflower:

**Comment by hiddeco** [comment]
This has been fixed in `v0.12.1`, released a couple of seconds ago.
